packages/gatsby-recipes/src/graphql-server/server.js

```diff
@@ -32,15 +32,19 @@ const pubsub = new PubSub()
 const PORT = process.argv[2] || 50400
 
 let lastState = {}
+let lastDone = 0
 const emitUpdate = state => {
   // eslint-disable-next-line no-unused-vars
   const { lastEvent, ...cleanedState } = state
-  if (!lodash.isEqual(cleanedState, lastState)) {
+  // isEqual doesn't handle values on objects in arrays 🤷‍♀️
+  const newDone = cleanedState.context.plan.filter(r => r.isDone).length
```

Comment:
Don't need to do this now, but this block of code might be useful to break out into a compare function next time we're in here so it reads more intuitively when figuring out if state/done has changed.

Reply:
done